src/scss/components/_list.scss

```diff
@@ -100,4 +109,97 @@
       padding-right: inherit;
     }
   }
+
+  &__title {
+    color: get-color(rastaGray);
+    display: block;
+    font-size: 12px;
```

Comment:
@felipeorlando can you validate this font-size with Bernardo? We are talking about increase our font sizes, because Google, on Lighthouse, is penalizing us because the fonts are too small on Gaiden (font-size > 16. [more info here](https://www.webpagetest.org/lighthouse.php?test=180125_YQ_20ac80da23cc2fc94b8e736800879ea5&run=1#seo))

Reply:
We will talk about the Lighthouse penalties in the future. 😉

Reply:
@felipeorlando I will create a issue about this and talk with our designers to fix it